src/main/java/org/myrobotlab/service/Python.java

```diff
@@ -270,7 +273,7 @@ public Object get(String pythonRefName) {
   Map<String, String> exampleFiles = new TreeMap<String, String>();
 
   transient LinkedBlockingQueue<Message> inputQueue = new LinkedBlockingQueue<Message>();
-  final transient InputQueue inputQueueThread;
+  transient InputQueue inputQueueThread;

```

Comment:
Should stay final, you can initialize it here even though it references `this`, like so:
```java
final transient InputQueue inputQueueThread = new InputQueue(this);
```